### DISCUSSION TITLE
Github issue or am i missing something?

### DISCUSSION THREAD

**Discussion by tristanrose** [community/community#46187] [Repositories] [answered]
### Select Topic Area

Question

### Body

So,  i rolled back to a previous commit in my private repo, using the `git reset --hard <commit>` in the cl which followed by letting me know it's now at the HEAD.  👍

followed by the `git push -f origin main` which triggered "Everything up-to-date" 👍

but my github repo didn't roll back to my current version on my local as i thought it was supposed to do with this commands and responses in cl.   ❌

is this a github issue or am i missing something? thanks

**Comment by SilasRodrigues19** [comment, marked as answer]
Hey @tristanrose 

Try this:

`git reset --hard <commit hash>`
`git push origin HEAD --force`

**Reply by tristanrose** [reply]
tried this.   unfortunately still the same.

do you think this repo being linked with Vercel is causing this repo not to dial back to specified commit hash?
or it should do it regardless and Vercel would just follow suit on the github roll back?

**Reply by tristanrose** [reply]
Hey @SilasRodrigues19

never-mind that worked -   i was still connected to my localhost server when doing it.

So i disconnected,  ran it again,  and it worked :)
